packages/evaluate/src/index.js

```diff
@@ -53,6 +59,12 @@ export const makeEvaluators = (makerOptions = {}) => {
       ...Object.getOwnPropertyDescriptors(endowmentState.endowments),
     });
 
+    // fullTransforms is an array of transforms. Each step takes a previous
```

Comment:
The repetition is on purpose?

Reply:
Yeah, I want the comments on both calls to reduce. Would it be better to have the long comment just the first time, and "see above" on the second one?

Reply:
that would be more my style, yes.

I spent quite a few minutes trying to figure out what was different between the two comments before I realized there wasn't anything different.